Objects/memoryobject.c

```diff
@@ -3255,6 +3255,9 @@ PyDoc_STRVAR(memory_f_contiguous_doc,
              "A bool indicating whether the memory is Fortran contiguous.");
 PyDoc_STRVAR(memory_contiguous_doc,
              "A bool indicating whether the memory is contiguous.");
+PyDoc_STRVAR(memory_exit_doc,
+             "__exit__($self, /, *exc_info)\n--\n\n"
+             "Release the underlying buffer exposed by the memoryview object.");
```

Comment:
```suggestion
PyDoc_STRVAR(memory_enter_doc,
             "__enter__($self)\n--\n\n"
             "Check the underlying buffer has not been released.");
PyDoc_STRVAR(memory_exit_doc,
             "__exit__($self, /, *exc_info)\n--\n\n"
             "Release the underlying buffer exposed by the memoryview object.");
```
(maybe the signature for __enter__ is not correct, it returns a reference to itself)

Reply:
It is rather the side effect.

Reply:
I left this for future. The `__enter__` methods either acquire the resource, in which case it should be explicitly documented, or do not have a side effect, in which case they do not need a docstring. I am not sure what is closer to this case.